cfgtool

```diff
@@ -268,17 +280,26 @@ And encrypted with the generated password.
 Please transmit the zip over a safe channel to it's user.
 
 In order to access the VPN the user must unpack the tar, make sure the 'nobody' group and user exists on their system and execute 'openvpn openvpn-client.cfg' inside the unpacked directory as root.
-  
+
 Stay safe. Use crypto. Don´t drink and factorize."
 
 }
 
+#Load a textfile filled with the emails of the new users
 load_clients() {
+  #Filename of the textfile with clients
   file_name="$1"; shift || fail "USAGE: ${script_file} load_clients <filename>"
+  #Read the text file line by line and add to array
   IFS=$'\n' clients=($(<$file_name))
   tmp_dir="./temp_clients"
-  rm -f "generated_clients.txt"
 
+  #Clean previous generated files
+  rm -rf "generated_clients.txt" $tmp_dir
+
+  #Rebuild the ca
+  yes YES | vpn_pki_setup
```

Comment:
++1! The intention here is to reset the entire configuration!
We might consider using the --no-regen-dh switch though!